tripleo/insights/io.py

```diff
@@ -52,6 +52,18 @@ class DeploymentOutline:
     """
 
 
+@dataclass
+class Topology:
+    """Description of the deployment's topology.
+    """
+    compute: int = 0
```

Comment:
There a few other possible node types for the topology. In my limited experience, I seen at least also freeipa and networking nodes. There is probably not enough types of nodes that we need to complicate excessively the desing of this class and the NodesInterpreter, but should be kept in mind that we will need to come back and add them at some point. I don't know if there is anywhere a document with all possible values.

Reply:
Yeah, I know that these are not all them. I gathered the types that I could from the 'nodes' files at Quickstart. If I do not have an example that gives me the keys to the type, there is not much that I can do here.

The good thing is that when more appear, it should not be difficult to extend this with so.

Reply:
A quick scan of the nodes folder of the triple-quicstart repo: https://opendev.org/openstack/tripleo-quickstart/src/branch/master/config/nodes shows that the only additional node type that we have not included is `CellController` ([link](https://opendev.org/openstack/tripleo-quickstart/src/commit/3728c672feafa6f1ed7b89ffc5a5f82eca641bb9/config/nodes/1ctlr_1cellctrl_1comp.yml#L36)) so it won't be a huge problem for us.

Reply:
Great, I seemed to have missed it. I will create a PR adding that option too.